HdfsAccessor.go

```diff
@@ -118,13 +118,14 @@ func (this *hdfsAccessorImpl) connectToNameNodeImpl(nnAddr string) (*hdfs.Client
 
 // Opens HDFS file for reading
 func (this *hdfsAccessorImpl) OpenRead(path string) (ReadSeekCloser, error) {
```

Comment:
This change is to reduce the number of connections on the name node. Add lock to limit parallelism (though it should be OK).